sway/input/cursor.c

```diff
@@ -787,7 +787,7 @@ static void handle_constraint_commit(struct wl_listener *listener,
 	check_constraint_region(cursor);
 }
 
-static void handle_request_set_cursor(struct wl_listener *listener,
+static void handle_pointer_set_cursor(struct wl_listener *listener,
```

Comment:
I'd prefer to keep "request" in the handler name here, to make it clear we react to a client request and not to a cursor change.